### ISSUE TITLE
Native mobile Polkadot app

### ISSUE THREAD

**Issue by CrispBull** [paritytech/trappist-extra#24]
Why not go native android or Kotlin Multiplatform instead of doing Flutter? Does this suggest it's possible to build a fully native decentralized app on Polkadot (not using Flutter/Dart)?

**Comment by Daanvdplas** [comment]
Yes you can! https://github.com/smol-dot/smoldot#can-i-embed-smoldot-into-a-mobile-application-or-an-application-in-general
